fhttp/http_utils.go

```diff
@@ -287,9 +287,9 @@ func generateSize(sizeInput string) (size int) {
 		}
 		size = s
 		log.Debugf("Parsed size %s -> %d", sizeInput, size)
-		if size > MaxPayloadSize {
-			log.Warnf("Requested size %d greater than max size %d, using max instead", size, MaxPayloadSize)
-			size = MaxPayloadSize
+		if size > fnet.MaxPayloadSize {
```

Comment:
move to a function so it can also be used in fortio_main to validate/set the size

Reply:
I didn't understand what you mean in this comment. The size is set according to input `sizeInput`

Reply:
The if is needed 3 times (not exceed max size(

Reply:
I got it now. I implemented the method in network instead of fortio_main. I can not check the size before `generateSize` method because, the input size is not flat. String checks are applied on `InputSize`.